src/Symfony/Component/Security/Core/Authorization/TraceableAccessDecisionManager.php

```diff
@@ -51,10 +52,25 @@ public function decide(TokenInterface $token, array $attributes, $object = null)
     {
         $result = $this->manager->decide($token, $attributes, $object);
 
+        $voterDetails = array();
+
+        foreach ($this->voters as $voter) {
```

Comment:
this breaks the lazy-loading of voters, as it will now load all voters even though the original code was not triggering their loading

Reply:
Arf, I didn't think about that at all... Thanks !